authz/rbac_translator.go

```diff
@@ -53,11 +56,23 @@ type rule struct {
 	Request request
 }
 
+type auditLogger struct {
+	Name       string
+	Config     structpb.Struct
```

Comment:
I'm guessing you chose to skip the json tags for these fields because they are single word names, and hence the json tag is obvious? It would still be nice to explicitly list the json tag for consistency.